### DISCUSSION TITLE
Custom Plugin for Youtube Series (commission)

### DISCUSSION THREAD

**Discussion by LowSoulGamers** [runelite/runelite#16921] [Ideas]
Howdy, (was advised to post this here by one of the mods in the RL discord)

I have a series idea that I have been dying to get started for a decent amount of time, but have never really dabbled in custom plugins. Would be looking to work with someone who has experience making custom plugins, can answer my (sometimes probably very stupid and obvious) questions, and would be able to make tweaks to the plugin as needed.

Don't want to give away tons of info, as I would like the idea to remain unique/secret until uploaded. Custom commission, so we can discuss payment after I tell you what my idea is/get an idea of how long it would take.

lowsoulgamers on Discord / https://twitter.com/Low_Soul_Gamers on twitter.

Thanks!

**Comment by digiholic** [comment]
Did anyone ever get back to you on this? It might be something within my wheelhouse.

**Reply by LowSoulGamers** [reply]
as of right now, yes
